astropy/coordinates/tests/test_regression.py

```diff
@@ -751,3 +751,9 @@ def test_regression_10291():
     assert_quantity_allclose(
         venus.separation(sun), 554.427 * u.arcsecond, atol=0.001 * u.arcsecond
     )
+
+
+def test_regression_17008():
```

Comment:
Can we please rename the test to be descriptive about its purpose and not start putting GitHub issue number in the names? I have been bitten before when a project moved from SVN to Git and all the numbers from Trac became useless on GitHub. Thanks!

Reply:
Will do! Or, at least, I'll add a proper docstring...

Reply:
OK, kept the function name the same since almost all in the file are like it, but added a docstring. Also felt that my other changes could use some more detail, so added that too. Thanks!

Reply:
Thanks! I will let coordinates people to do the rest of the review. Does look reasonable to me but I am no expert in coordinates.

Reply:
I think we need `utils.iers`, but that's mostly me. Though @eteq and @taldcroft did quite a bit on it too - either of you have a bit of time? It is reasonably straightforward...

Reply:
It would be much better to have a test for many different attributes and methods of an empty `SkyCoord`, not just `itrs`. There's a few checks in [`test_sky_coord.test_ops()`](https://github.com/astropy/astropy/blob/fe293ffa1a844e51be31abab3e25e3762b451ba4/astropy/coordinates/tests/test_sky_coord.py#L788), but bugs like this prove we need more.

Alternatively there could be tests for each method or attribute that would check empty, scalar and array `SkyCoord` instances.

That being said, if the bug being fixed here is about the `itrs` property in particular then it's fine to implement in this pull request a new regression test for just this one attribute.

Reply:
Really, the bug is in `iers`, where it is now fairly fully tested. I included this regression test just because I think it makes sense to always include a test of the actual example provided in the original issue. 

So, it is not specific to the `.itrs` property, but to `iers`. I don't really want to expand tests to other `SkyCoord` attributes - that way my bug fixing rate will rapidly shrink to zero...